core/src/components/modal/test/swipe-to-close/e2e.ts

```diff
@@ -0,0 +1,11 @@
+import { testModal } from '../test.utils';
+
```

Comment:
The `swipe-to-close` tests are not necessary. We have tests that do the same thing in the `spec` directory.